torch/csrc/distributed/rpc/script_functions.cpp

```diff
@@ -50,6 +50,30 @@ c10::intrusive_ptr<c10::ivalue::Future> rpcTorchscriptCall(
   return futPtr;
 }
 
+PyRRef remoteTorchscript(
```

Comment:
Any reason one function is called `rpcTorchscript*Call*` and another is called `remoteTorchscript`?

Reply:
I'll remove the word, "Call", to make it in align with function names in "python_functions.h".